src/scaffoldmaker/meshtypes/meshtype_3d_wholebody2.py

```diff
@@ -0,0 +1,432 @@
+                    (19, [[8.459, -0.683, 0.298], [0.258, -0.010, 0.370], [0.003, 0.175, 0.003], [-0.005, 0.000, -0.009], [-0.127, 0.001, 0.089], [0.036, -0.004, -0.000]]),
+                    (20, [[8.601, -0.685, 0.694], [0.024, 0.006, 0.399], [-0.001, 0.175, -0.002], [-0.002, -0.000, 0.003], [-0.099, -0.000, 0.006], [0.077, 0.003, -0.084]]),
+                    (21, [[8.100, 0.665, 0.000], [0.775, 0.027, 0.275], [-0.005, 0.175, -0.002], [-0.022, -0.000, -0.006], [-0.048, 0.000, 0.136], [-0.157, 0.006, -0.039]]),
+                    (22, [[8.459, 0.683, 0.298], [0.258, 0.010, 0.370], [-0.000, 0.175, -0.005], [0.005, 0.000, 0.009], [-0.127, 0.002, 0.089], [0.034, 0.004, 0.001]]),
+                    (23, [[8.601, 0.685, 0.694], [0.024, -0.006, 0.399], [0.001, 0.175, 0.002], [-0.004, 0.000, -0.003], [-0.104, 0.000, 0.006], [0.071, -0.009, -0.087]])
+                ]]),
+            "userAnnotationGroups": [
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '1-3',
+                    'name': 'head',
+                    'ontId': 'None'
+                },
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '6-8',
+                    'name': 'torso',
+                    'ontId': 'None'
+                },
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '4,11-13, 5,14-16',
+                    'name': 'arms',
+                    'ontId': 'None'
+                },
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '9,17-18-19, 10,20-21-22',
+                    'name': 'legs',
+                    'ontId': 'None'
+                }
+            ]
+        })
+
+
+class MeshType_3d_wholebody2(Scaffold_base):
+    """
+    Generates a 3-D hermite bifurcating tube network, with linear basis through wall.
+    """
+
+    @staticmethod
+    def getName():
+        return "3D Whole Body 2"
+
+    @classmethod
+    def getParameterSetNames(cls):
+        return [
+            "Default",
+            "Human 1",
+            "Human 2 Coarse",
+            "Human 2 Medium",
+            "Human 2 Fine"
+        ]
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName="Default"):
+
+        options = {
+            'Base parameter set': parameterSetName,
+            "Network layout": getDefaultNetworkLayoutScaffoldPackage(cls, parameterSetName),
+            "Number of elements around head": 12,
+            "Number of elements around torso": 12,
+            "Number of elements around arms": 8,
+            "Number of elements around legs": 8,
```

Comment:
arms -> arm, legs -> leg. For consistency with terms.

Reply:
Done.